hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/upgrade/UpgradeTask.java

```diff
@@ -77,6 +80,9 @@ public UpgradeTask(ConfigurationSource config,
   }
 
   public CompletableFuture<UpgradeManager.Result> getUpgradeFutureByVolume() {
+    final File volumeUpgradeLockFile =
```

Comment:
Please move this statement into the supplyAsync scope.

Reply:
> Please move this statement into the supplyAsync scope.

`File volumeUpgradeLockFile` it will be used in whenComplete.